repository_service_tuf_api/api/targets.py

```diff
@@ -48,3 +48,23 @@ def delete(
     _user=Security(validate_token, scopes=[SCOPES_NAMES.delete_targets.value]),
 ):
     return targets.delete(payload)
+
+
+@router.post(
+    "/publish/",
+    summary=(
+        "Submit a task to publish targets."
+        f"Scope: {SCOPES_NAMES.write_targets.value}"
+    ),
+    description=(
+        "Trigger a task to publish targets not yet published from the "
+        "RSTUF Database"
```

Comment:
When I read this description I asked myself: how will the user know who is the current batch of non-published targets?

Reply:
Yes, there is no way to know what is not published. The user can control it by themself from:
- the requests and the [status of the tasks](https://github.com/vmware/repository-service-tuf-worker/issues/190) ([if the targets are published in the same request](https://github.com/vmware/repository-service-tuf-worker/issues/175#issuecomment-1410644318))
- if they don't publish in the same request (above), by using this endpoint, we are implementing

I think it is more about documenting in the Guide: 
- https://github.com/vmware/repository-service-tuf/issues/227

Reply:
That's okay if I just added a batch of targets without publishing them, but if someone is serious about this feature and wants big batches of targets before publishing them it's not scaling well.
Do you think we need an endpoint or other way to show this?
Maybe we can mention other non-published changes - added or removed targets in the response to a user's requests to add or remove targets without publishing them?

Reply:
> That's okay if I just added a batch of targets without publishing them, but if someone is serious about this feature and wants big batches of targets before publishing them it's not scaling well.
> Do you think we need an endpoint or other way to show this?

In that case, We don't need an endpoint for it. We should give an additional optional parameter in the payload for this method/endpoint, passing which delegated roles they want to publish. I don't think we should go deep on it now. It can be done in another interaction upon request as a new feature or feature enhancement.

> Maybe we can mention other non-published changes - added or removed targets in the response to a user's requests to add or remove targets without publishing them?

We do it for the add/removing targets. We return the published targets.
Doing this control will impact the performance computation, as we must interact in the middle. Let's start simple, and we can improve if there is a user case/request for it.